src/test/scala/io/github/sentenza/hacktoberfest18/sort/SortTests.scala

```diff
@@ -0,0 +1,151 @@
+package io.github.sentenza.hacktoberfest18.sort
+import io.github.sentenza.hacktoberfest18.util.ArrayUtil
+import org.scalatest.FunSuite
+import util.Random
+import io.github.sentenza.hacktoberfest18.sort.CommonSortings.CommonSorting._
+
+/**
+  * This class is a test suite for the methods in object CommonSortings. To run
+  * the test suite, you can either:
+  *  - run the "test" command in the SBT console
+  *  - right-click the file in zour editor of choice and 'run as test' or 'run'
+  */
+class SortTests extends FunSuite {
```

Comment:
I'm afraid but I cannot accept a duplicate of what we have already under `test`. If you want to enhance what's already in there you should add more test cases.